source/includes/_sub_category_search_metrics.md

```diff
@@ -0,0 +1,57 @@
+# Sub Category Search Metrics
```

Comment:
We have a fairly consistent, yet undocumented format for this project.  And unfortunately some work is needed to use this format.

I suggest simply copying a good example like [this](https://github.com/GannettDigital/api-docs/blob/master/source/includes/_gannett_display_report.md) and changing it for your needs.

And I see you don't require a gmaid.  This is a REALLY bad idea.  You could pull it off, but you will be swimming upstream against our codebase.  Everything is based around advertisers.  For example, most of our API consumers are not client center.  For them, each request needs to verify that the caller has access to that advertiser.  Of course, you might say just don't bother with the verification.  There are cases where we do this, but I don't feel this is called for here.  Make your life easier and require a gmaid.  Otherwise there is a lot of code your estimate is depending on inheriting that won't work.

In your documentation I expect to see things like what parameters does your API accept and a best effort attempt at describing what they are.

The parameter is business_sub_category_id, not business_sub_category.

What does your API do if it doesn't receive a **business_sub_category**?  What if the the **business_sub_category** can't be found.  Are business sub categories unique across platforms?  I don't see a platform here.  One way to solve this is to instead use '**USA_1234**' to denote **business_category_id** **1234** for **platform_id** 1.

Reply:
I hate to ask, but if there is a sub_category is there a category?  Should that be in the response also?  Will your API need to return multiple sub-categories or all?

Would this API be more useful to a caller if they could pass an array of **global_master_campaign_id** values?

As written, how will client center know the **business_category_id** to pass?

Reply:
great question, actually that is *not* resolved. i was building on what was available in the schema. what other data tables / schemas available for this? for example, is there another table called `BusinessCategories` where these sub business categories can be linked up to? 

I will bring this up with product, feels like i am holding on to a one piece of information but missing the larger picture... Thanks for review and giving me insights with the API return and such @brianroth-rl